src/components/GeneralInfo/GeneralInformation/GeneralInformation.js

```diff
@@ -10,13 +10,13 @@ import { SortByDirection } from '@patternfly/react-table';
 
 import { systemProfile } from '../../../store/actions';
 import InfoTable from '../InfoTable';
-import OperatingSystemCard from '../OperatingSystemCard';
-import SystemCard from '../SystemCard';
-import BiosCard from '../BiosCard';
-import InfrastructureCard from '../InfrastructureCard';
-import ConfigurationCard from '../ConfigurationCard';
-import SystemStatusCard from '../SystemStatusCard';
-import DataCollectorsCard from '../DataCollectorsCard/DataCollectorsCard';
+import { OperatingSystemCard } from '../OperatingSystemCard';
```

Comment:
If it is required to use the named exports, let's add a comment here to indicated that and why.

I can easily see myself going over this in the future and using the default exports again. :D